### DISCUSSION TITLE
Is Album disk version in Ampache6  = MusicBrainz release disambiguation notes?

### DISCUSSION THREAD

**Discussion by SimonHova** [ampache/ampache#3413] [Ideas]
All of my music is perfectly tagged in MusicBrainz; when I saw this feature on the roadmap, it looks like it lines up perfectly with the feature. I know that in recent builds, artists in Ampache now have an **Update details from MusicBrainz** option in addition to the **Update from tags** button.

Are there plans to put a similar feature for the albums (which would be very cool, btw), and if so, does that field fill from the album disambiguation field in MusicBrainz?

**Comment by lachlan-00** [comment]
that would be good to add. It would mix in with the change to subtitle/version/etc in the wiki https://github.com/ampache/ampache/wiki/ampache6-details#album-disk-versionsubtitles-added-to-the-grouping

you also reminded me that i didn't put that new field in the edit window.

**Comment by SimonHova** [comment]
Yes, please! Also, while we're on the topic, can I also add the feature request to optionally paste the disambig notes into the subsonic artist/album/title field? I'm a heavy Subsonic user, and I'll often have to flip through four or five tracks with identical names and different lengths, it'll be helpful if I could have the option to append the notes to show if the tracks were recorded live, or were explicit, etc.

**Reply by lachlan-00** [reply]
i will definitely be making that part available as an option like the original year title options. many people will have things that look the same but are different versions. especially with album covers that look the same it's impossible to tell without extra info.

with songs i think there will be ways to add extra infor based on a release date/ recording date that doesn't match the original recording date but not going deep onto songs until i get the album parts right.
